### ISSUE TITLE
No entry bin errors and dedi crashing - Jul11 Apex update

### ISSUE THREAD

**Issue by NomadVR** [acemod/ACE3#4087]
**Arma 3 Version:** `1.62` Apex day (stable)
**CBA Version:** `v2.4.1.160606` (stable)
**ACE3 Version:** `3.6.0.0` (stable)

**Mods:**
- `@CBA_A3`
- `@ace`
It's specifically Ace, I tried it just using CBA and didn't seem to get the same issue.

**Description:**
- When I get to the main menu I get the error:
No entry 'bin\config.bin/CfgWeapons/ItemInfo.scope'.
I noticed someone mentioned I needed AceX. Tried it and got this error:
No entry 'bin\config.bin/CfgWeapons.Single'.

**Steps to reproduce:**
- Loading to the main menu or into the arsenal.

**Where did the issue occur?**
- Dedicated and locally. Locally I can still run the game after closing the error. But it makes my dedi shutdown upon loading the mission and shows the same error in the chatbox.

**Placed Modules:**
- None.

**RPT log file:**
- Dedi server rpt: http://pastebin.com/Ry4fbwPM

**Comment by jonpas** [comment]
ACE3 has been updated to 3.6.0, which is 1.62 Apex compatible - https://github.com/acemod/ACE3/releases/tag/v3.6.1

**Comment by NomadVR** [comment]
You mean 3.6.1?

**Comment by jonpas** [comment]
Oh yes, my bad haha, fixed. :)

**Comment by esteldunedain** [comment]
> You mean 3.6.1?

Yes

**Comment by NomadVR** [comment]
I was using ACE via workshop. Not sure if it's been updated there yet but I'll grab this one that you linked. Thanks for the fast reply!

**Comment by jonpas** [comment]
Indeed, Workshop has not been updated yet.

**Comment by NomadVR** [comment]
Still getting the same errors. Does CBA need an update as well?

**Comment by jonpas** [comment]
Can you upload an RPT with just CBA and ACE3 without any other mods? Above RPT has other mods loaded as well.

**Comment by NomadVR** [comment]
I managed to get everything working. The issue was that I dragged the new ace pbos into the old ones and didn't remove the others that were mixed in from the old version when I tested just CBA+ACE. The errors I'm experiencing are caused by pretty much every other mod. One of which is also messing with the ACE UI on the main menu to make it how it used to be before it got updated for Apex. Not sure what's causing that yet.
